manifests/golang.yml

```
@@ -343,7 +343,7 @@ tests/:
         gin: v1.37.0
     test_asm_standalone.py:
       Test_AppSecStandalone_UpstreamPropagation: v1.71.0
-      Test_AppSecStandalone_UpstreamPropagation_V2: missing_feature
+      Test_AppSecStandalone_UpstreamPropagation_V2: v1.73.0
```

```suggestion
      Test_AppSecStandalone_UpstreamPropagation_V2: v1.73.0-dev
```

`-dev` is the suffix we use on main for the system-tests to have a base tag to refer to